environment.yml

```diff
@@ -17,9 +17,5 @@ dependencies:
   - numba
   - pip
   - pip:
-    - git+https://github.com/ExecutableBookProject/MyST-NB.git
-    - jupyter-cache[cli]
-    - git+https://github.com/mwouts/jupytext.git
     - git+https://github.com/ExecutableBookProject/jupyter-book
```

Comment:
@najuzilu should this be listed here given we have an action step to install `jupyter_book` I think that is good to have as it will show in the logs what versions of underlying software are installed.

Reply:
I will leave it in there as a record of dependency. It looks like the latest master is installed on the nightly builds in quantecon-mini-example

Reply:
@mmcky isn't this information available when `jupyter-book` is installed? The last line after a conda install is a list of all packages and their respective versions.

Reply:
Ah my page did not reflect your latest comment @mmcky. If you think this is best, that's fine with me.